src/Microsoft.AspNetCore/WebHost.cs

```diff
@@ -149,12 +149,21 @@ private static IWebHost StartWith(string url, Action<IServiceCollection> configu
         /// <returns>The initialized <see cref="IWebHostBuilder"/>.</returns>
         public static IWebHostBuilder CreateDefaultBuilder(string[] args)
         {
-            var builder = new WebHostBuilder()
-                .UseKestrel((builderContext, options) =>
+            var builder = new WebHostBuilder();
+
+            if (string.IsNullOrEmpty(builder.GetSetting(WebHostDefaults.ContentRootKey)))
+            {
+                builder.UseContentRoot(Directory.GetCurrentDirectory());
+            }
+            if (args != null)
```

Comment:
Why did you move this?

Reply:
Only for clarity so all of the config would be together, it's not a functional change.